src/Validators/TextPolicyValidator.php

```diff
@@ -16,13 +16,14 @@ class TextPolicyValidator {
 	private $badWords;
 	private $whiteWords;
 
-	const CHECK_URLS = 1;
-	const CHECK_BADWORDS = 4;
-	const IGNORE_WHITEWORDS = 8;
+	public const CHECK_URLS = 1;
+	// TODO: Update these once a replacement for blacklist/whitelist is decided https://phabricator.wikimedia.org/T254646
+	public const CHECK_BADWORDS = 4;
+	public const IGNORE_WHITEWORDS = 8;
```

Comment:
For a safe transition, you can already add `CHECK_DENIED_WORDS` and `IGNORE_ALLOWED_WORDS` with the same int values, marking the others as deprecated. Once the FF code uses the new constants, we can remove the deprecated ones. Thanks for paying attention to language!